.github/workflows/comment-stalled.yml

```diff
@@ -0,0 +1,22 @@
+name: Comment on issues and PRs when labelled stalled
+on:
+  issues:
+    types: [labeled]
+  pull_request:
+      types: [labeled]
```

Comment:
This one will always fail for PRs in this repo because `GITHUB_TOKEN` doesn't have the appropriate permissions

Reply:
Good catch, thx! Was not supposed to be part of that commit actually, remains from some testing I did before pushing.